### DISCUSSION TITLE
Zephyr cannot pass the uart_pl011 test suite on FVP(fvp_base_revc_2xaemv8a).

### DISCUSSION THREAD

**Discussion by cocoeoli** [zephyrproject-rtos/zephyr#46019] [Q&A]
I want to test the official uart tests suite(The suit locate at  zephyr/tests/driver/uart/uart_basic_api) on arm fvp(The board is fvp_base_revc_2xaemv8a ), but i can't pass the suit below:
![image](https://user-images.githubusercontent.com/40254934/170291182-936aa741-b9ce-40ce-8f91-557c68987e13.png)
And when I test the same code with qemu_cortex_a53, it can work well. So I debug the code and find it block on below code(I thinks it means that my uart interrupt work wrong here):
![image](https://user-images.githubusercontent.com/40254934/170298223-8980096a-8edd-4601-9874-c839e9759b37.png)
Finnially, I find that it is because the system can not receive the uart interrupt  signal and keep the data_received flag false all the time. So I want to know is there any addition configuration I need to do on fvp to make the uart interrupt signal route to my system and make tests work well?
Below is my  board.camke file for fvp para:
`-C bp.secure_memory=0 -C bp.refcounter.non_arch_start_at_default=1 -C bp.pl011_uart0.out_file=- -C bp.pl011_uart0.unbuffered_output=1 -C bp.terminal_0.start_telnet=1 -C bp.vis.disable_visualisation=1
-C bp.vis.rate_limit-enable=0 -C gic_distributor.ARE-fixed-to-one=1 -C gic_distributor.ITS-device-bits=16`
And my uart&console related config para is below:
`CONFIG_SERIAL=y`
`CONFIG_UART_PL011=y`
`CONFIG_UART_PL011_PORT0=y`
`CONFIG_UART_INTERRUPT_DRIVEN=y`
`CONFIG_CONSOLE=y`
`CONFIG_UART_CONSOLE=y`
Thanks all of you for your response!

**Comment by ayankuma** [comment]
@cocoeoli Do you know the command you have used ?

**Reply by cocoeoli** [reply]
Thanks, I have passed the test.
